src/app/app.ts

```diff
@@ -13,6 +13,7 @@ app.use(menuRouter.routes()).use(menuRouter.allowedMethods())
 
 app.use(async (ctx: Koa.Context, next: () => Promise<any>)=>{
     try {
+        console.log(ctx.URL)
```

Comment:
Adding console.log doesn't look like refactoring ;) Only if you add some meaningful message to it and pretend that it is logging (you don't have logging library yet, right?).